lib/adroll/api/ad.rb

```diff
@@ -7,7 +7,7 @@ class Ad < AdRoll::Api::Service
                           :headline, :body, :message, :headline_dynamic,
                           :body_dynamic, :message_dynamic, :is_fb_dynamic,
                           :dynamic_template_id, :product, :logo, :background,
-                          :call_to_action, :multiple_products]
+                          :call_to_action, :multiple_products, :tracking]
```

Comment:
Can you just whitelist everything in the doc? Just don't want to have to come back and do this again anytime soon.